config/nginx-config/default-pages/50x.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>50x Error</title>
+<style>
+    body {
+        width: 80%;
+        margin: 0 auto;
+        font-family: Tahoma, Verdana, Arial, sans-serif;
+        background-color: #2b303b;
+        color: #c0c5ce;
+    }
+    pre, code {
+        background: #2b303b;
+        border-radius: 4px;
+        border: 1px solid #4f5b66;
+    }
+    code {
+        color: #a3be8c;
+        padding: .1em .2em;
+    }
+    .box {
+        border-radius: 4px;
+        padding: 1.5em;
+        background: #343d46;
+        box-sizing: border-box;
+        margin-bottom: 1em;
+        margin-top: 1em;
+    }
+    h1 {
+        color:white;
+    }
+</style>
+</head>
+<body>
+  <div class="box">
+    <h1>An error occurred in VVV.</h1>
```

Comment:
```suggestion
    <h1>You encountered a 50X error!</h1>
    <p>Nginx tried to serve the page, but encountered an error code beginning with 50</p>
```

Not all 500 errors are because of VVV